addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.py

```diff
@@ -15,7 +15,7 @@ def default_get(self, fields_list):
         result = super().default_get(fields_list)
         active_model = self._context.get('active_model')
         active_ids = self._context.get('active_ids')
-        if len(active_ids) > 1:
+        if active_ids and len(active_ids) > 1:
```

Comment:
what about a default value in the `.get` above?
`self._context.get('active_ids', [])`

Reply:
yes, we can do that too :+1: